packages/apps/graph/src/devnet/safe-transfer.ts

```diff
@@ -1,22 +1,15 @@
 import type { ICommandResult } from '@kadena/client';
-import { Pact } from '@kadena/client';
+import { Pact, createSignWithKeypair } from '@kadena/client';
+import { submitClient } from '@kadena/client-utils/core';
 import { PactNumber } from '@kadena/pactjs';
 import type { ChainId } from '@kadena/types';
-import { devnetConfig } from './config';
+import { dotenv } from '@utils/dotenv';
 import type { IAccount } from './helper';
-import {
-  inspect,
-  listen,
-  logger,
-  sender00,
-  signAndAssertTransaction,
-  stringifyProperty,
-  submit,
-} from './helper';
+import { logger, sender00, stringifyProperty } from './helper';
 
 export async function safeTransfer({
```

Comment:
Good for now, I will we add safeTransfer to the client-utils as well.

Reply:
That would be awesome!